### PR TITLE
USWDS-Site: Documentation updates for release 3.7.0

### PR DESCRIPTION
**Pull request by amyleadem**
# Summary

Changelogs for all updates in the USWDS 3.7.0 release.

## Preview links
- [Banner changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.7.0/components/banner/#changelog)
  - PR:  https://github.com/uswds/uswds/pull/5551
  - Changelog: https://github.com/uswds/uswds-site/pull/2301
  - PR: https://github.com/uswds/uswds/pull/5542
  - Changelog: https://github.com/uswds/uswds-site/pull/2327
- [Card changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.7.0/components/card/#changelog)
  - PR: https://github.com/uswds/uswds/pull/5571
  - Changelog: https://github.com/uswds/uswds-site/pull/2317
- [Date picker changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.7.0/components/date-picker/#changelog)
  - PR: https://github.com/uswds/uswds/pull/5374
  - Changelog: https://github.com/uswds/uswds-site/pull/2218
- [Footer changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.7.0/components/footer/#changelog)
  - PR: https://github.com/uswds/uswds/pull/5588
  - Changelog: https://github.com/uswds/uswds-site/pull/2328
- [Icon changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.7.0/components/icon/#changelog)
  - PR: https://github.com/uswds/uswds/pull/5589
  - Changelog: https://github.com/uswds/uswds-site/pull/2329
- [Identifier changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.7.0/components/identifier/#changelog)
  - PR: https://github.com/uswds/uswds/pull/5491
  - Changelog: https://github.com/uswds/uswds-site/pull/2266
- [Modal changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.7.0/components/modal/#changelog)
  - PR:  https://github.com/uswds/uswds/pull/5315
  - Changelog: https://github.com/uswds/uswds-site/pull/2117
- [Range slider changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.7.0/components/range-slider/#changelog)
  - PR:  https://github.com/uswds/uswds/pull/5413
  - Changelog: https://github.com/uswds/uswds-site/pull/2220
  - PR: https://github.com/uswds/uswds/pull/5472
  - Changelog: https://github.com/uswds/uswds-site/pull/2310
- [Settings changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.7.0/documentation/settings/#changelog)
  - PR: https://github.com/uswds/uswds/pull/5582
  - Changelog: https://github.com/uswds/uswds-site/pull/2313
- [Site alert changelog](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/release-3.7.0/components/site-alert/#changelog)
  - PR: https://github.com/uswds/uswds/pull/5550
  - Changelog: https://github.com/uswds/uswds-site/pull/2324

### CODE REVIEW

**Comment by amyleadem** [comment]
@thisisdano I confirmed the following items: 
- [x] Confirmed that all 3.7.0 items have related documentation updates
- [x] Confirmed that changelogs are written in past tense and have periods.
- [x] Confirmed changelog information is accurate.
- [x] Confirmed that the changelog item tags are appropriate
- [ ] Confirmed that changelog dates and version numbers have been updated
    - Not done yet
- [ ] Confirmed that all references to node version 16.18.0 have been updated to 20.9.0
    - Added [a comment](https://github.com/uswds/uswds-site/pull/2332/files#r1383763469) about a possible edit to the README

File: .nvmrc
[the remark this answers is not in the document]
Comment (mahoneycm) [reply .nvmrc:1]: I think it'd be a great idea to update there as well. Should it be contained in a separate PR or can we add it here?

**Review by mejiaj — APPROVED** [review]
Content looks good and all changes seem to be captured.

**Review by mahoneycm — APPROVED** [review]

File: .nvmrc
[the remark this answers is not in the document]
Comment (amyleadem) [reply .nvmrc:1]: Updated in [1515116](https://github.com/uswds/uswds-site/pull/2332/commits/15151160c8e6b523180864de1380402a6b029c9e)

File: _data/changelogs/component-card.yml [hunk 1]
```diff
@@ -2,6 +2,13 @@ title: Card
 type: component
 changelogURL:
 items:
+  - date: 2023-11-9
```
Comment (amyleadem) [inline _data/changelogs/component-card.yml:5]: Other release dates on the 9th of the month have a leading zero 

```suggestion
  - date: 2023-11-09
```

Comment (thisisdano) [reply _data/changelogs/component-card.yml]: Done!

**Pull request merged**